helmfile.d/snippets/derived.gotmpl

```diff
@@ -87,4 +87,5 @@ environments:
             enabled: false #{{ eq $provider "aws" }}
         otomi:
           version: {{ $otomiTag }}
-        versions: {{- $versions | toYaml | nindent 10 }} 
+        versions: {{- $versions | toYaml | nindent 10 }}
+      - ../core.yaml
```

Comment:
could you clarify why core.yaml is now needed?

Reply:
I moved it to the next phase as it is not user input